File: Great Outdoor/Great Outdoor Back End/GreatOutdoor/src/main/java/com/cg/go/model/CartModel.java
package com.cg.go.model;

public class CartModel {

	
	private long cartId;
	private String userId;
	private static double cartItemPrice;
	private long quantity;
	private ProductModel product;
	
	public long getCartId() {
		return cartId;
	}
	public void setCartId(long cartId) {
		this.cartId = cartId;
	}
	public String getUserId() {
		return userId;
	}
	public void setUserId(String userId) {
		this.userId = userId;
	}
	public static double getCartItemPrice() {
		return cartItemPrice;
	}
	public static void setCartItemPrice(double cartItemPrice) {
		CartModel.cartItemPrice = cartItemPrice;
	}
	public long getQuantity() {
		return quantity;
	}
	public void setQuantity(long quantity) {
		this.quantity = quantity;
	}
	public ProductModel getProduct() {
		return product;
	}
	public void setProduct(ProductModel product) {
		this.product = product;
	}
	
	
	
}

File: Great Outdoor/Great Outdoor Front End/Great Outdoor UI/project/src/app/model/cart.model.ts
// model for cart
export class Cart {

    constructor(public cartId: number, public userId: string, public productId: string,
        public cartItemPrice: number, public quantity: number) {

        this.cartId = cartId;
        this.userId = userId;
        this.productId = productId;
        this.cartItemPrice = cartItemPrice;
        this.quantity = quantity;

    }
}